### PR TITLE
ZH-trans: add  2016-07-00-Citrix-Netscaler-And-Kubernetes.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by tengqm** [comment]
@xichengliudui Please avoid approve PRs submitted by yourself. Even if this operation is allowed by the platform, it is not a good community practice. You may want to ping other reviewers to review it, you can bring it up in the team meetings, you can try persuade other reviewers how important/urgent the PR is, but ... you should not approve it by yourself.

**Comment by xichengliudui** [comment]
I saw the robot assigned to me `If they are not already assigned, you can assign the PR to them by writing /assign @xichengliudui in a comment when ready.` so I approved.

File: content/zh/blog/_posts/2016-07-00-Citrix-Netscaler-And-Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+-->
+
+1. Citrix 的客户希望从 NetScaler 获得与他们在迁移到云计算时相同的第4层到第7层功能，因为他们开始使用 Kubernetes 部署他们的容器和微服务体系结构&nbsp;
+2. Kubernetes 为运行带有自动化工作负载交付的容器和 vm 提供了一个经过验证的基础设施 
+3. NetScaler CPX 为日志和分析平台 [NetScaler 管理和分析系统](https://www.citrix.com/blogs/2016/05/24/introducing-the-next-generation-netscaler-management-and-analytics-system/) 提供了第4层到第7层的服务和高效的遥测数据。
+
+<!--
+I wish all our experiences working together with a technical partner were as good as working with GCP. We had a list of issues to enable our use cases and were able to collaborate swiftly on a solution. To resolve these, GCP team offered in depth technical assistance, working with Citrix such that NetScaler CPX can spin up and take over as a client-side proxy running on each host.&nbsp;  
+-->
+我希望我们所有与技术合作伙伴一起工作的经验都能像与 GCP 一起工作一样好。我们有一个问题列表来支持我们的用例，并且能够在解决方案上快速协作。为了解决这些问题，GCP 团队提供了深入的技术支持，与 Citrix 合作，这样 NetScaler CPX 就可以在每个主机上作为客户端代理运行。
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/blog/_posts/2016-07-00-Citrix-Netscaler-And-Kubernetes.md]: > 这样 NetScaler CPX 就可以在每个主机上作为客户端代理运行。

从而使得 NetScaler CPX 能够在每台主机上作为客户端代理启动运行。

File: content/zh/blog/_posts/2016-07-00-Citrix-Netscaler-And-Kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+-->
+接下来，需要在 GCP 入口负载均衡器的数据路径中插入 NetScaler CPX，使 NetScaler CPX 能够将流量分散到前端 web 服务器。NetScaler 团队进行了修改，以便 NetScaler CPX 监听 API 服务器事件，并配置自己来创建 VIP、IP 表规则和服务器规则，以便跨前端应用程序接收流量和负载均衡。谷歌云平台团队提供反馈和帮助，验证为克服技术障碍所做的修改。完成了!
+
+<!--
+NetScaler CPX use case is supported in [Kubernetes 1.3](https://kubernetes.io/blog/2016/07/kubernetes-1.3-bridging-cloud-native-and-enterprise-workloads). Citrix customers and the broader enterprise market will have the opportunity to leverage NetScaler with Kubernetes, thereby lowering the friction to move workloads to the cloud.&nbsp;  
+-->
+NetScaler CPX 用例支持 [Kubernetes 1.3](https://kubernetes.io/blog/2016/07/kubernets-1.3 - bridge -cloud-native-and-enterprise-workload)。Citrix 的客户和更广泛的企业市场将有机会利用 NetScaler 和 Kubernetes，从而降低将工作负载转移到云计算的阻力。&nbsp;
```
Comment (tengqm) [inline content/zh/blog/_posts/2016-07-00-Citrix-Netscaler-And-Kubernetes.md]:
```suggestion
NetScaler CPX 用例在 [Kubernetes 1.3](https://kubernetes.io/blog/2016/07/kubernets-1.3 - bridge -cloud-native-and-enterprise-workload) 中提供支持。Citrix 的客户和更广泛的企业市场将有机会基于 Kubernetes 享用 NetScaler 服务，从而降低将工作负载转移到云平台的阻力。&nbsp;
```

File: content/zh/blog/_posts/2016-07-00-Citrix-Netscaler-And-Kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+<!--
+You can learn more about&nbsp;NetScaler CPX [here](https://www.citrix.com/networking/microservices.html).  
+-->
+您可以[在此处](https://www.citrix.com/networking/microservices.html)了解有关 NetScaler CPX 的更多信息。
```
Comment (tengqm) [inline content/zh/blog/_posts/2016-07-00-Citrix-Netscaler-And-Kubernetes.md]:
```suggestion
您可以在[此处](https://www.citrix.com/networking/microservices.html)了解有关 NetScaler CPX 的更多信息。
```

File: content/zh/blog/_posts/2016-07-00-Citrix-Netscaler-And-Kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+<!--
+_&nbsp;-- Mikko Disini, Director of Product Management - NetScaler, Citrix Systems_
+-->
+_&nbsp;-- Citrix Systems NetScaler 产品管理总监 Mikko Disini
```
Comment (tengqm) [inline content/zh/blog/_posts/2016-07-00-Citrix-Netscaler-And-Kubernetes.md]:
```suggestion
_&nbsp;-- Mikko Disini，Citrix Systems NetScaler 产品管理总监
```

**Comment by xichengliudui** [comment]
已更新。

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @xichengliudui, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/123) has no assignee. will not be reported to the task issues

**Pull request merged**